internal/cp/proxy_factory_test.go

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cp
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestObjectNameForProxy(t *testing.T) {
```

Comment:
Could you collect these tests in a single umbrella test like the one for atomic long?

Reply:
Added at https://github.com/hazelcast/hazelcast-go-client/pull/901/commits/628c21c86995619aae201315bf28b9aef639102a.